test/testdrive/get-started.td

```diff
@@ -71,7 +71,7 @@ id true bigint
 
 
 > CREATE MATERIALIZED VIEW winning_bids AS
-  SELECT * FROM highest_bid_per_auction WHERE extract(epoch FROM end_time) * 1000 < mz_logical_timestamp()
+  SELECT * FROM highest_bid_per_auction WHERE end_time < mz_logical_timestamp()
```

Comment:
Maybe convert from `mz_logical_timestamp()` to `mz_now()`?

Reply:
Done.